lib/cli.js

```diff
@@ -158,6 +150,22 @@ module.exports = function broccoliCLI(args) {
   return actionPromise || Promise.resolve();
 };
 
+function setEnvironment(options) {
+  // Proxy aliases
+  if (options.prod) {
+    options.environment = 'production';
+  } else if (options.dev) {
+    options.environment = 'development';
+  }
+
+  // If environment manually supplied via CLI set BROCCOLI_ENV for broccoli-stew and broccoli-env compat
+  if (options.environment) {
+    process.env.BROCCOLI_ENV = options.environment;
+  } else {
+    options.environment = 'development';
```

Comment:
Shouldn't this default to `process.env.BROCCOLI_ENV || 'development'`?

Reply:
As mentioned, I don't know if we should proxy `process.env.BROCCOLI_ENV` into `options.env`.
The intention here is to ensure that `BROCCOLI_ENV` alone will continue to work with `broccoli-env/stew`, and `broccoli-env/stew` will work with `--environment`. By allowing `BROCCOLI_ENV` to flow into `options.env` we're saying `you don't need to use --environment, you can use BROCCOLI_ENV alternatively`. Is that something we want?